site/guide/set-up-model-workflows.qmd

```diff
@@ -0,0 +1,213 @@
+    - To edit a step, hover over the step until the **{{< fa ellipsis-vertical >}}** menu appears. Then click on it and select **{{< fa gear >}} Configure**.
+
+5. After you're finished with step configuration, click **Save** to apply your changes before closing the configuration panel with **{{< fa x >}}**. 
+
+    ![](configure-steps.gif){fig-alt="A gif workflow step configuration."}
+
+    After you've configured a step, you can then [link your workflow together](#link-workflow-together).
+
+::: {.callout-tip appearance="minimal"}
+Refer to **[Available step types](#available-step-types)** for information on available step types and details on how to set up conditional requirements.
+:::
+
+### Available step types
+
+#### Status Change
+
+- Transitions a status. 
+- Requires [available resource statuses](customize-resource-statuses.qmd) associated with the workflow.
+
+| Field | Description |
+|---:|---|
+| When these conditions are met (optional) | [Add conditional requirements](#add-conditional-requirements) to qualify for this step.  |
+| Set status to (required) | Select the status to transition to. |
+: Status Change step configuration {.hover tbl-colwidths="[40,60]"}
+
+#### User Action
+- Creates a button that performs an action on behalf of the user. 
+- Requires [available resource statuses](customize-resource-statuses.qmd) associated with the workflow.
+
+| Field | Description |
+|---:|---|
+| Display action button when (optional) | [Add conditional requirements](#add-conditional-requirements) for displaying this step.  |
+| Action button label (required) | Text that displays on your action button. |
+| Request fields (optional) | Include [custom fields](manage-inventory-custom-fields.qmd) alongside your action if desired. Select the field, then click **+ Add Field** to insert.  |
+| To set status (required) | Select the [status](customize-resource-statuses.qmd) to transition to. |
+: User Action step configuration {.hover tbl-colwidths="[40,60]"}
+
+::: {.grid}
+
+::: {.g-col-7}
+![](configure-user-action.png){fig-alt="A screenshot showing a User Action step configuration with a request field of Date On Hold."}
+:::
+
+::: {.g-col-5}
+![](user-action-custom-field.png){fig-alt="A screenshot showing the action transition panel with the Date On Hold custom field."}
+
+::: {.callout-tip title="Example" appearance="minimal"}
+A `Put On Hold` action that requests a value in the Date On Hold field, and transitions the resource status to On Hold.
+:::
+
+:::
+
+:::
+
+#### Approval
+- When your resource reaches the approval step, users in the selected Approval Group will receive your configured message. 
+- Requires a User type [custom field](manage-inventory-custom-fields.qmd) to select from, and must be connected to both an approval and rejection **[Status Change](#status-change)** step.
```

Comment:
```suggestion
- Requires a **User** type [custom field](manage-inventory-custom-fields.qmd) to select from, and must be connected to both an approval and rejection **[Status Change](#status-change)** step.
```

I think that is the actual field name based on the link, so it should be bolded?